src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -316,7 +288,156 @@ To ensure your CSS files are accessible, add the `crossorigin="anonymous"` attri
+                        errorSamplingRate
+                        maskTextSelector
+                        enabled
+                        collectFonts
+                        inlineImages
+                        inlineStylesheet
+                        samplingRate
+                        maskAllInputs
+                        maskInputOptions {
+                          color
+                          date
+                          datetimeLocal
+                          email
+                          month
+                          number
+                          range
+                          search
+                          select
+                          tel
+                          text
+                          textArea
+                          time
+                          url
+                          week
+                        }
+                      }
+                    }
+                    errors {
+                      field
+                      errorClass
+                      description
+                    }
+                  }
+                }
+                ```
+            </td>
+            <td>
+                ```json
+                {
+                  "sessionReplay": {
+                    "enabled": true,
+                    "collectFonts": true,
+                    "blockSelector": "String",
+                    "maskTextSelector": "String",
+                    "inlineStylesheet": false,
+                    "maskAllInputs": true,
+                    "maskInputOptions": {
+                      "color": true,
+                      "date": true,
+                      "textArea": true
+                    },
+                    "samplingRate": 9.99,
+                    "errorSamplingRate": 1.11
+                  }
+                }
+                ```
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## Data consumption [#data-consumption]
+
+### View consumption [#view-consumption]
+
+Between April 8 and May 8, 2024, while session replay bytes are free, you can view your consumption to help with planning once the bytes become paid:
+    1. Go to <DoNotTranslate>**Application Settings**</DoNotTranslate>.
+    2. Scroll down to <DoNotTranslate>**Session replay**</DoNotTranslate>.
+    3. In the <DoNotTranslate>**Session sample size**</DoNotTranslate> section, click <DoNotTranslate>**View your account's consumption**</DoNotTranslate>.
+
+    <img
+        title="View account consumption on application settings page"
+        alt="screenshot of application settings page"
+        src={browserViewConsumptionDetails}
+    />
+
+### Plan for paid consumption [#plan-for-paid-consumption]
+
+After May 8, 2024, session replay will follow the same consumption pricing as your other browser bytes. The amount of bytes produced depends on the count, length, and user-activity levels of sessions, as well as the complexity of your site's DOM.
```

Comment:
I assume there's a plan to remove the April and May dates after we pass May 8?

Reply:
Yes, I've got a followup ticket for this!